.eslintrc

```diff
@@ -0,0 +1,213 @@
+ecmaFeatures:
```

Comment:
Ok, done, and put up the info on setting up ESLint on Slack. Let me know if you have any other questions. Thanks for the PR with the fixed config, @cgreene!